x-pack/plugins/observability_solution/dataset_quality/public/components/common/insufficient_privileges.tsx

```diff
@@ -54,7 +54,10 @@ export const PrivilegesWarningIconWrapper = ({
 }) => {
   const [isPopoverOpen, togglePopover] = useToggle(false);
 
-  const handleButtonClick = useCallback(() => togglePopover(true), [togglePopover]);
+  const handleButtonClick = useCallback(
+    () => togglePopover(!isPopoverOpen),
+    [isPopoverOpen, togglePopover]
+  );
```

Comment:
Caught a bug where user was not able to close the warning explanation on the privileges

https://github.com/user-attachments/assets/2a8b08a6-6954-4cc4-8bc4-50b70d790060